app/api/resources/grpc_client.py

```diff
@@ -148,3 +149,12 @@ def get_dataset_attributes(
                 genome_uuid=genome_uuid, dataset_type=dataset_type
             )
         return self.stub.GetAttributesValuesByUUID(dataset_attributes)
+
+    def get_genome_by_specific_keyword(self, assembly_accession_id: str):
```

Comment:
It seems that you are looking up genome id specifically by assembly accession id. If so, you probably shouldn't say `by_specific_keyword` in the function name, and in the class name in `models/genome.py`.

Both thoas and the grpc service allow various fields to be used as a "keyword"; but here you are only using a specific one.